templates/tsConstructs/constructor.ejs

```diff
@@ -4,11 +4,11 @@
 if(signatures && signatures[0]){
     if (signatures[0].type.type === 'reference') {
     %>
-    <h4 class="code-ref"><%-name%> (<%- include('commaSeperatedParams', {'params': signatures[0].parameters}); %>): <a href="#<%-signatures[0].type.name%>"><%-signatures[0].type.name%></a></h4>
+    <h4 class="code-ref"><%-name%>( <%-include('commaSeperatedParams', {'params': signatures[0].parameters});%> ) : <a href="#<%-signatures[0].type.name%>"><%-signatures[0].type.name%></a></h4>
     <%
     } else {
     %>
-    <h4 class="code-ref"><%-name%> (<%- include('commaSeperatedParams', {'params': signatures[0].parameters}); %>): <%-signatures[0].type.name%></h4>  
+    <h4 class="code-ref"><%-name%>( <%-include('commaSeperatedParams', {'params': signatures[0].parameters});%> ) : <%-signatures[0].type.name%></h4>  
```

Comment:
IIUC, this will emit `Ctor( a, b, c )` and `Ctor(  )`. I personally find this formatting unusual - our coding style is `Ctor(a, b, c)` and `Ctor()`.

What is the benefit of this change?

(The same comment applies to the change on L7 too.)